src/api/subscriptions.js

```diff
@@ -278,11 +278,16 @@ export const SubscriptionFlows = {
  *   force redirect flow for any window.open operation, including payments.
  *   The "auto" value either uses a redirect or a popup flow depending on
  *   what's possible on a specific environment. Defaults to "auto".
- *
+ * - enableSwgAnalytics - if set to true then events logged by the publisher's
+ *   client will be sent to Google's SwG analytics service.  This information is
+ *   used to compare the effectiveness of Google's buy-flow events to those
+ *   generated by the publisher's client code.  This includes events sent to
+ *   both propensity and logger.
  * @typedef {{
  *   experiments: (!Array<string>|undefined),
  *   windowOpenMode: (!WindowOpenMode|undefined),
  *   analyticsMode: (!AnalyticsMode|undefined),
+ *   enableSwgAnalytics: (boolean|undefined),
```

Comment:
Should this be (!boolean|undefined)? I don't know if this matters or not.

Reply:
boolean is a non-nullable value so no.

Boolean is nullable so it would need the !